src/systems/performer_detector/PerformerDetector.cc

```diff
@@ -109,7 +114,8 @@ void PerformerDetector::PostUpdate(
 
   // Double negative because AxisAlignedBox does not currently have operator+
   // that takes a position
-  auto region = this->detectorGeometry - (-modelPose.Pos());
+  auto region = this->detectorGeometry -
+    (-(modelPose.Pos() + this->poseOffset.Pos()));
```

Comment:
This implies that the position offset is given in world frame. If we want it to be relative relative to the model, I think this should be
```suggestion
    (-(modelPose.Pos() + modelPose.Rot() * this->poseOffset.Pos()));
```

Reply:
Thanks: e34d3bda28d79ebbbc82a8d473e0452b2069971f